Enhance dashboard configuration documentation (#153)
microsoft/aspire.dev: src/frontend/src/content/docs/dashboard/configuration.mdx

The dashboard is configured when it starts up. Configuration includes frontend and OpenTelemetry Protocol (OTLP) addresses, the resource service endpoint, authentication, telemetry limits, and more.

<Aside type="caution">
The dashboard displays information about resources, including their configuration, console logs and in-depth telemetry.

Data displayed in the dashboard can be sensitive. For example, secrets in environment variables, and sensitive runtime data in telemetry. Care should be taken to configure the dashboard to secure access.

<LearnMore>
For more information, see [dashboard security](/dashboard/security-considerations/).
</LearnMore>
</Aside>

## How to configure the dashboard

> How you configure the dashboard depends on whether it's being started by the Aspire AppHost project, or whether it is started in [standalone mode](overview#standalone-mode).

Author: ```suggestion
How you configure the dashboard depends on whether it's started by the Aspire AppHost project or run in [standalone mode](overview#standalone-mode).
```

### Aspire AppHost

> The AppHost automatically configures the dashboard. However, it is possible to override values if required. The recommeded way to configure the dashboard from the Aspire AppHost is to add environment variables to the _launchSettings.json_ file.

Author: ```suggestion
The AppHost automatically configures the dashboard, but you can override values if needed. The recommended way to configure the dashboard from the Aspire AppHost is by adding environment variables to the _launchSettings.json_ file.
```

```json title="JSON — launchSettings.json" {14-15}
{
  "$schema": "https://json.schemastore.org/launchsettings.json",
  "profiles": {
    "https": {
      "commandName": "Project",
      "dotnetRunMessages": true,
      "launchBrowser": true,
      "applicationUrl": "https://localhost:17134;http://localhost:15170",
      "environmentVariables": {
        "ASPNETCORE_ENVIRONMENT": "Development",
        "DOTNET_ENVIRONMENT": "Development",
        "ASPIRE_DASHBOARD_OTLP_ENDPOINT_URL": "https://localhost:21030",
        "ASPIRE_RESOURCE_SERVICE_ENDPOINT_URL": "https://localhost:22057",
        "DASHBOARD__TELEMETRYLIMITS__MAXLOGCOUNT": "50000"
      }
    }
  }
}
```

> The proceeding launch settings increases `Dashboard:TelemetryLimits:MaxLogCount` [telemetry limit](#telemetry-limit) to 50,000. The `:` delimiter must be replaced with double underscore (`__`) in environment variable names.

Author: ```suggestion
These launch settings increase the `Dashboard:TelemetryLimits:MaxLogCount` [telemetry limit](#telemetry-limit) to 50,000. The `:` delimiter must be replaced with double underscore (`__`) in environment variable names.
```
